CONTRIBUTING.md

```diff
@@ -169,10 +169,14 @@ Environment variables:
 - You **need** to set ESX and either VM_IP (in which case we'll use 1 VM) or
 both VM1 and VM2 environment variables
 
+- You need to set `DOCKER_HUB_REPO` as environment variable to complete our move to use [managed plugin](https://github.com/vmware/docker-volume-vsphere/blob/master/plugin/Makefile#L30).
```

Comment:
Please add ***The build will use your user name (the output of 'whoami`) to decide on the DockerHub repo name. 
If you want to use another DockerHub repo*** you need to set ...